webpack/components/SCCProductPage/index.js

```diff
@@ -0,0 +1,18 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as actions from './SCCProductPageActions';
+import SCCProductPage from './SCCProductPage';
+import * as Selector from './SCCProductPageSelectors';
+
+// map state to props
+const mapStateToProps = (state) => ({
+  scc_products: Selector.selectSCCProducts(state),
+  scc_account_id: Selector.selectSCCAccountId(state),
+});
+
+// map action dispatchers to props
+const mapDispatchToProps = (dispatch) => bindActionCreators(actions, dispatch);
+
+// export connected component
+export default connect(mapStateToProps, mapDispatchToProps)(SCCProductPage);
```

Comment:
we used this pattern quite a lot in the past,
though today it's better to `useDispatch` directly from `react-redux`

Reply:
Do you have an example for this? Not sure how it should look like.

Reply:
Yeah, here's a simple case: https://github.com/theforeman/foreman/blob/4bcd67ddb14abad4c9b44e213e19e1dfda7c8b7e/webpack/assets/javascripts/react_app/components/ConfirmModal/index.js